flexget/plugins/input/filesystem.py

```diff
@@ -156,7 +163,7 @@ def get_folder_objects(self, folder, recursion):
         else:
             return folder.walk(errors='ignore')
 
-    def get_entries_from_path(self, path_list, match, recursion, test_mode, get_files, get_dirs, get_symlinks):
+    def get_entries_from_path(self, path_list, match, match_reject, skip_hidden_files, recursion, test_mode, get_files, get_dirs, get_symlinks):
```

Comment:
This line is too long. Couldn't some of those arguments be set as class attributes instead?